### ISSUE TITLE
Add `!important` to focus outline

### ISSUE THREAD

**Issue by FlyersPh9** [iTwin/stratakit#894]
We've seen too many instances of products removing the focus outline which is a serious A11y violation.  Let's try to override their overrides by applying `!important`.

Comment from @mayank99:
> We'll need to test out what happens with iTwinUI theme bridge before we release this change.
